app/controllers/admins/users_controller.rb

```diff
@@ -12,10 +12,17 @@ def index
 
     def edit
       @user = User.find(params[:id])
+      render 'edit'
```

Comment:
remove this string, action render template with the same name automatically.  you should add `render 'template_name'` if you use another name